server/bundles/io.cloudbeaver.service.auth/schema/service.auth.graphqls

```diff
@@ -81,17 +81,25 @@ type AuthProviderInfo {
     required: Boolean!
 }
 
-type AuthInfo {
-    redirectLink: String
 
+type AuthInfo {
+    redirectLink: String @deprecated
 
     authId: String @deprecated
 
-    authStatus: AuthStatus!
+    authStatus: AuthStatus! @deprecated
 
     userTokens: [UserAuthToken!]
 }
 
+type AsyncAuthInfo @since(version: "25.0.3") {
+    redirectLink: String!
```

Comment:
i think it can be optional because there is not always a link?

Reply:
I am not sure @alexander-skoblikov

Reply:
right now it's always not null, because AsyncAuth = federated auth